_sections/80-update-my-project.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Update
+fa-icon: arrow-circle-up
+bg: '#49a7e9'
+color: 'white'
+style: center
+---
+
+# Update your project to be Python 3 only
```

Comment:
I'd call this section something like 'Practicalities' - but it may be easiest to leave it for a future PR.